roles/edpm_frr/molecule/default/converge.yml

```diff
@@ -30,6 +31,10 @@
       ansible.builtin.include_role:
         name: "osp.edpm.edpm_frr"
         tasks_from: "download_cache.yml"
+    
```

Comment:
We probably need to remove it from https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci/playbooks/molecule-test.yml#L4 and  add `gather_facts: false` in molecule prepare/converge.yaml for all roles rather than collecting and then cleaning them. That way all roles would be tested without default `gather_facts: true` I guess.

Reply:
I looked at disabling it in `test_deps`:
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/molecule/common/test_deps/tasks/main.yml#L17-L41

Seemed like there was more thought required before doing that though. Since `test_deps` gather them, I think everything else will already have a fact cache available because they all call `test_deps` first.

But maybe we just need to disable it everywhere and then fix whatever breaks.

Reply:
Ah, right. Yeah I see, it's specifically excluding `min`.

Reply:
Ok, so I'll drop the meta task to clear facts and we can set a Depends-On for your PR in CIFMW:
https://github.com/openstack-k8s-operators/ci-framework/pull/2383

Reply:
I've changed it in all roles with https://github.com/openstack-k8s-operators/edpm-ansible/pull/767/files.

Reply:
Ah cool, we can close this one then.